src/libraries/System.Console/src/System/Console.cs

```diff
@@ -515,19 +539,22 @@ public static void SetCursorPosition(int left, int top)
             }
         }
 
+        [UnsupportedOSPlatform("android")]
         [UnsupportedOSPlatform("browser")]
         public static bool TreatControlCAsInput
         {
             get { return ConsolePal.TreatControlCAsInput; }
             set { ConsolePal.TreatControlCAsInput = value; }
         }
 
+        [UnsupportedOSPlatform("android")]
         [UnsupportedOSPlatform("browser")]
         public static Stream OpenStandardInput()
         {
             return ConsolePal.OpenStandardInput();
         }
 
+        [UnsupportedOSPlatform("android")]
```

Comment:
What about `SetIn`, `Read()` and `ReadLine()`? Those are all annotated as `[UnsupportedOSPlatform("browser")]`, should they also have `[UnsupportedOSPlatform("android")]` on them?

Reply:
Addressed, thanks